### DISCUSSION TITLE
making the RabbitMQ.ServiceModel work in .NET core

### DISCUSSION THREAD

**Discussion by birojnayak** [rabbitmq/rabbitmq-dotnet-client#1278] [Ideas]
Hi Guys,
We are enabling RabbitMQ  support in CoreWCF repo natively. I am wondering if I can make the  [RabbitMQ.ServiceModel](https://github.com/rabbitmq/rabbitmq-dotnet-client/tree/v4.0.1/projects/wcf/RabbitMQ.ServiceModel/src/serviceModel) also compatible with latest .net core (.net 6) under rabbitmq-dotnet-client. 

Let me know your thoughts !
Biroj.

**Comment by lukebakken** [comment]
You are welcome to add that code to your project. If you haven't noticed already, it was removed in this PR - https://github.com/rabbitmq/rabbitmq-dotnet-client/pull/248

We do not wish to have that code in this project because we don't want to maintain it!

**Comment by birojnayak** [comment]
@lukebakken I didn't know you folks are not maintaining it... I am talking with owner of WCF and CoreWCF how we can own this..

**Reply by lukebakken** [reply]
Yes, that's why the code was removed several years ago. It's all open-source so please feel free to copy the code from this repository's history into the CoreWCF project.
